pkg/cmd/pr/view/view.go

```diff
@@ -101,6 +103,14 @@ func viewRun(opts *ViewOptions) error {
 		return utils.OpenInBrowser(openURL)
 	}
 
+	opts.IO.StartProgressIndicator()
+	reviews, err := api.ReviewsForPullRequest(apiClient, repo, pr)
```

Comment:
1. Should this be under `if opts.Comments`?
2. Since both reviews and comments are paginated collections, could we query them in parallel instead of sequentially?

Reply:
1. We display the reviews (without bodies) with the other metadata at the top of the pull request even without the `--comments` flag so we need to retrieve them outside of the `opts.Comments` block.
2. That is a good improvement.